src/hooks/useFetch.js

```diff
@@ -0,0 +1,257 @@
+import { useEffect, useContext } from 'react';
+import { ytapikey } from '../utils/config';
+import { GlobalContext } from '../context/GlobalContext';
+
+export const useFetch = (videoId) => {
```

Comment:
We talked about this earlier today, in the next delivery this should be returning data from the API instead of the mocked data, if you need help ping me in slack